src/main/java/com/iexec/core/task/listener/TaskListeners.java

```diff
@@ -111,6 +111,12 @@ public void onPleaseUploadEvent(PleaseUploadEvent event) {
         log.info("NotifyUploadingWorker completed[uploadingWorkerWallet={}]", workerWallet);
     }
 
+    @EventListener
+    public void onResultUploadTimeoutEvent(ResultUploadTimeoutEvent event) {
```

Comment:
What's the purpose of this listener?

Reply:
I just kept it in case we need to do some handling or send notifications (...). But I can remove it if you want currently it does not have any relevant purpose (except for logging)